src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -84,6 +84,7 @@ export class ChoiceGroup extends React.Component<IChoiceGroupProps, IChoiceGroup
                                 onBlur={this._onBlur.bind(this, option)}
                             />
                             {this._renderField(option)}
+                            {this._renderContentBelowOption(option, keyChecked)}
```

Comment:
What does this do exactly?

Reply:
Renders additional content below the ChoiceGroup option
http://take.ms/5snkw

Reply:
Ok, looks nice! 👍